index.html

```diff
@@ -280,7 +323,7 @@ <h3 class="more-info">
                 </ul>
                 <div class="vector-form"></div>
             </section>
-        </footer>        
+        </footer>
```

Comment:
- [ ] [OPTIONAL]: You should consider using icons instead of images for your social media links, some of them have a low resolution. Also, as icons, you are able to personalize them (changing colors, for instance), to match your design better. I recommend using [font awesome](https://fontawesome.com/).